### ISSUE TITLE
Can't nest routes under home path anymore in 2.7?

### ISSUE THREAD

**Issue by danconnell** [emberjs/ember.js#14073]
https://ember-twiddle.com/8627acbb0ddd9593e20ad67e9a721941?openFiles=router.js%2C&route=%2Ftest2

Given these 2 routes:

```
  this.route('test1', {
    path: '/'
  }, function() {
   this.route('test2'); 
  });
  this.route('misc', {
    path: '/:param'
  });
```

If you visit the URL /test2, the misc route is used instead of test1/test2

**Comment by rwjblue** [comment]
This seems like a variation of https://github.com/emberjs/ember.js/issues/13960.

/cc @mixonic @nathanhammond

**Comment by nathanhammond** [comment]
Confirmed, this is an issue.

**Comment by rwjblue** [comment]
Ember 2.7.2 was released to fix this issue.

@danconnell - Can you confirm this is fixed?

**Comment by danconnell** [comment]
Confirmed, this is fixed in 2.7.2. Thanks!
